app/views/govuk_publishing_components/components/docs/show_password.yml

```
@@ -0,0 +1,79 @@
+name: Show password input
+description: A password input field that allows the password to be shown.
+body: |
+  Adds a password reveal button to an input that toggles its type from password to text, revealing the password. Does not appear if JS is disabled.
```

Optinionated nitpick - JavaScript > JS:

```suggestion
  Adds a password reveal button to an input that toggles its type from password to text, revealing the password. Does not appear if JavaScript is disabled.
```

---

Since I thrive on nitpicks, I can't ignore this one. Changed (and the other one).